CHANGELOG.md

```diff
@@ -18,12 +18,7 @@ _We tried to deprecate every major change, resulting in practically no breakage
 - Every aspect of Agents.jl is orthogonal to `AgentBasedModel`: movement and neighbor searching in any space, data collection, visualizations, etc., are independent of the specific type of `AgentBasedModel` and work out of the box with any model.
 - Logic of when to collect data in `run!` has been improved to accommodate both discrete and continuous time models. This is reflected in the new options for the keyword `when`.
   - A new keyword `init` is now available for `run!` to data collect from the model before evolving it. Whether data were collected at time 0 or not was not really obvious in the original version of `run!` due to the ambiguity of the previous handling of `when`.
-- A new `@multiagent` macro allows to run multi-agent simulations much more efficiently. It has
-  two version: In `:opt_speed` the created agents are optimized such as there is virtually
-  no performance difference between having 1 agent type at the cost of each agent occupying
-  more memory that in the `Union` case. In `:opt_memory` each agent is optimized to occupy practically
-  the same memory as the `Union` case, however this comes at a cost of performance versus having 1 type.
-- `@multiagent` kinds support multiple dispatch like syntax with the `@dispatch` macro.
+- Integration with `DynamicSumTypes.@sumtype` macro allows to run multi-agent simulations much more efficiently.
```

Comment:
You can't alter a previous changelog entry, that defeats the whole point of making a changelog. You need a new entry for the new version.

Reply:
yes sorry